src/test/ui/async-fn-path-elision.rs

```diff
@@ -0,0 +1,16 @@
+// edition:2018
```

Comment:
This test should be moved to a dedicated folder for async/await; having it here makes reviewing the stabilization report harder to review and also clutters the already polluted `ui/` folder.

Reply:
There is no such folder today. The existing async fn tests live in `ui/async-fn-*.rs`. I'd prefer not to change all of that in this PR.

Reply:
OK; can you make a C-cleanup issue for that?

Reply:
https://github.com/rust-lang/rust/issues/60418